compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/NodeFactory.java

```diff
@@ -512,21 +512,6 @@ public static BracedExpressionNode createBracedExpressionNode(
         return stBracedExpressionNode.createUnlinkedFacade();
     }
 
-    /**
-     * @deprecated Use {@link #createCheckExpressionNode(SyntaxKind, Token, ExpressionNode, OnFailCheckNode)} instead.
-     */
-    @Deprecated
```

Comment:
These deprecated APIs are not supported by our treegen implementation. Therefore, please undo deprecated api changes when committing.

Reply:
Updated [8a51481](https://github.com/ballerina-platform/ballerina-lang/pull/41098/commits/8a514812a9aa20e6f470e046c943e55c47d782f5)